packages/gatsby-cli/src/reporter/index.js

```diff
@@ -382,14 +403,16 @@ const reporter: Reporter = {
         return reporter.panic(...args)
       },
       done: () => {
+        updateProgress(true)
         span.finish()
         reporterActions.endActivity({
           id,
           status: ActivityStatuses.Success,
         })
       },
       set total(value) {
-        reporterActions.setActivityTotal({ id, total: value })
+        unflushedTotal = value
```

Comment:
i think the set total should also reset the `unflushedProgress` to zero?

Reply:
i did open a PR #19874

Reply:
(As explained in a comment on that PR, I don't think that to be the case)